addon/templates/components/lt-body.hbs

```diff
@@ -56,7 +56,7 @@
       </table>
 
       {{#if onScrolledToBottom}}
-        {{lt.infinity rows=rows onScrolledToBottom=onScrolledToBottom scrollBuffer=scrollBuffer}}
+        {{lt.infinity rows=rows onScrolledToBottom=onScrolledToBottom scrollableContent=".lt-scrollable"}}
```

Comment:
> Lastly, this PR is not working in Safari (using rAF) b/c the scrollBuffer is 500 (> than the height of the scrolling area). If the scrollBuffer was like 100, this would work across browsers.

Reply:
We could.  I assumed this was for `vertical-collection`, so it would change things up quite a bit in that case.  Looks like a scroll buffer of 100 adds quite a bit more rows to the vertical collection scenario...